sdk/src/Services/S3/Custom/Model/CreatePresignedPostRequest.cs

```diff
@@ -0,0 +1,66 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Amazon.Runtime;
+using Amazon.Util;
+
+namespace Amazon.S3.Model
+{
+    /// <summary>
+    /// Container for the parameters to create a presigned POST request for S3.
+    /// </summary>
+    public class CreatePresignedPostRequest : AmazonWebServiceRequest
+    {
+        /// <summary>
+        /// Gets or sets the name of the S3 bucket for the presigned POST.
+        /// </summary>
+        public string BucketName { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the key (name) of the object for the presigned POST.
+        /// </summary>
+        public string Key { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the expiration time for the presigned POST.
+        /// </summary>
+        public DateTime? Expires { get; set; }
+        
+        /// <summary>
+        /// Gets or sets additional form fields to include in the presigned POST.
+        /// </summary>
+        public Dictionary<string, string> Fields { get; set; }
+        
+        /// <summary>
+        /// Gets or sets the policy conditions for the presigned POST.
+        /// </summary>
+        public List<S3PostCondition> Conditions { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the CreatePresignedPostRequest class.
+        /// </summary>
+        public CreatePresignedPostRequest() 
+        {
+            Expires = AWSSDKUtils.CorrectedUtcNow.AddHours(1);
```

Comment:
Why are we defaulting to an hour? Seems like this should be null requiring users to decide what the expiration should be and give it to us.

Reply:
same with python https://github.com/boto/botocore/blob/1bcde58b83bff9dfc2d81bd42a796aa069a0975b/botocore/signers.py#L671

Reply:
Okay then make sure the documentation for the `Expires` property says this defaults to an hour from when the `CreatePresignedPostRequest` instance was created.

Reply:
updated comment in bcd54bc5d15143e54a4789dab1f6c4bd4e00ad08